cardano-cli/test/cardano-cli-golden/Test/Golden/Shelley/Genesis/InitialTxIn.hs

```diff
@@ -5,14 +5,15 @@ module Test.Golden.Shelley.Genesis.InitialTxIn where
 import           Test.Cardano.CLI.Util
 
 import           Hedgehog (Property)
+import qualified Hedgehog.Extras as H
 import qualified Hedgehog.Extras.Test.Golden as H
 
 {- HLINT ignore "Use camelCase" -}
 
 hprop_golden_shelleyGenesisInitialTxIn :: Property
 hprop_golden_shelleyGenesisInitialTxIn = propertyOnce $ do
   verificationKeyFile <- noteInputFile "test/cardano-cli-golden/files/input/shelley/keys/genesis_verification_keys/genesis-utxo.vkey"
-  goldenUtxoHashFile <- noteInputFile "test/cardano-cli-golden/files/golden/shelley/keys/genesis_utxo_hashes/utxo_hash"
+  goldenUtxoHashFile <- H.note "test/cardano-cli-golden/files/golden/shelley/keys/genesis_utxo_hashes/utxo_hash"
```

Comment:
Did you intentionally not change golden -> input here?

Reply:
Yes because `goldenUtxoHashFile` is not used as an input file.  The code before was misleading.